package.json

```diff
@@ -185,7 +185,7 @@
 		"@microsoft/dev-tunnels-ssh-keys": "^3.11.8",
 		"@microsoft/dev-tunnels-ssh-tcp": "^3.11.8",
 		"@segment/analytics-node": "^1.0.0-beta.24",
-		"configcat-node": "^8.0.0",
+		"configcat-node": "^9.1.1",
```

Comment:
Let's update in another PR or pushing directly right now it shows me some method is deprecated 
![image](https://github.com/gitpod-io/gitpod-vscode-desktop/assets/25115070/93d32077-e4ae-4524-9006-1a467c76fb93)

Reply:
I'll downgrade to 8.0.0